osu.Game/Rulesets/Mods/ModSuddenDeath.cs

```diff
@@ -17,4 +21,26 @@ public abstract class ModSuddenDeath : Mod
         public override bool Ranked => true;
         public override Type[] IncompatibleMods => new[] { typeof(ModNoFail), typeof(ModRelax), typeof(ModAutoplay) };
     }
-}
\ No newline at end of file
+
+    public abstract class ModSuddenDeath<T> : ModSuddenDeath, IApplicableToRulesetContainer<T>, IApplicableToScoreProcessor
+        where T : HitObject
+    {
+        private ScoreProcessor scoreProcessor;
+
+        public void ApplyToRulesetContainer(RulesetContainer<T> rulesetContainer)
+        {
+            rulesetContainer.OnJudgement += judgement =>
+            {
+                if (judgement.Result == HitResult.Miss)

```

Comment:
I think this conditional also has to have `|| scoreProcessor.Combo.Value == 0`, no? I.e. sliderbreaks should fail?